README.md

```diff
@@ -22,6 +20,34 @@ You can still raise issues or pull requests if you want to. As soon as we’re a
 
 [Try out the examples!](https://alphagov.github.io/accessible-autocomplete/examples/)
 
+---
+
+## Support
+
+The GOV.UK Design System team maintains the accessible autocomplete as a standalone component. However, we’re only able to put in minimal work to maintain and support it.
```

Comment:
We're waiting for broader feedback from the team, however with this wording you might be able to avoid repeating "maintain":

```suggestion
The GOV.UK Design System team maintains the accessible autocomplete as a standalone component. However, we’re only able to put in minimal work to support it.
```

Reply:
Thanks! Will make the same change in the PR for the country-picker README.